SU-HSDO/suhumsci
# STN-480:  Fix spacing beside navigation dropdown toggle

# READY FOR REVIEW

## Summary
The work in this PR prevents navigation dropdown arrow icons from overlapping the text when the child items are private pages.

Issue: JIRA [STN-480](https://sparkbox.atlassian.net/browse/STN-480)

## Need Review By (Date)
8/13/2020

## Urgency
medium

## Steps to Test
1. In your CLI, run `npm run test` and confirm all tests pass.
2. In order to test this PR you may need to do a database sync with staging. If you do not have a link to `Menu Arrow Example` within the main navigation of your local sparkbox sandbox site, you will need to do a database sync.
3. Visit https://sparkbox-sandbox-stage.stanford.edu/ and note the overlapping icon on `Menu Arrow Example` in the main navigation.
4. Go to http://sparkbox-sandbox.suhumsci.loc/ and confirm the icon beside `Menu Arrow Example` no longer overlaps the text.
5. Test logged in and logged out states to make sure the icon is spaced correctly
